pkgs/data/fonts/camingo-code/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchurl, unzip }:
+
+stdenv.mkDerivation rec {
+  name = "camingo-code-${version}";
+  version = "1.0";
+
+  src = fetchurl {
+    url = https://github.com/chrissimpkins/codeface/releases/download/font-collection/codeface-fonts.zip;
+    sha256 = "1gbpfa5mqyhi5yrb6dl708pggiwp002b532fn3axiagb0cxxf02s";
+  };
+
+  buildInputs = [ unzip ];
+
+  installPhase = ''
+    mkdir -p $out/share/fonts/truetype
+    mkdir -p $out/share/doc/${name}
+    cp -v camingo-code/*.ttf $out/share/fonts/truetype/
+    cp -v camingo-code/*.txt $out/share/doc/${name}/
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = https://www.myfonts.com/fonts/jan-fromm/camingo-code/;
+    description = "CamingoCode is a monospaced typeface designed for source-code editors";
```

Comment:
Per the guidelines, the description shouldn't contain the package name. I realize this is kind of a grey area, as the font name may be of particular interest, but in general it would belong in a `longDescription`; any description of the form "X is a Foo" is usually better as simply "A Foo".